tests/IntegrationTests/WebAppUiTests/WebAppUiTests.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="$(MicrosoftAspNetCoreMvcTestingVersion)" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNetTestSdkVersion)" />
+    <PackageReference Include="Microsoft.Playwright" Version="1.36.0" />
```

Comment:
I will add this version to the relevant props file if one exists at the UI tests level